src/commands/functions-artifacts-setup-cleanup-policy.ts

```diff
@@ -0,0 +1,191 @@
+import * as clc from "colorette";
+
+import { Command } from "../command";
+import { FirebaseError } from "../error";
+import { needProjectId } from "../projectUtils";
+import { confirm } from "../prompt";
+import { requirePermissions } from "../requirePermissions";
+import { requireAuth } from "../requireAuth";
+import { logBullet, logSuccess } from "../utils";
+import * as artifactregistry from "../gcp/artifactregistry";
+import * as artifacts from "../functions/artifacts";
+
+/**
+ * Command to set up a cleanup policy for Cloud Run functions container images in Artifact Registry
+ */
+export const command = new Command("functions:artifacts:setup-cleanup-policy")
```

Comment:
Did we actually get approval to be the first command to use kebab-case? Also why setup-cleanup-policy and not set-cleanup-policy? This can be used post-setup.

Reply:
Oh I think the reviewers missed this - I didn't realize that there was no precedent for kebab-case.

Reply:
Resolved to rename the command `functions:artifacts:setpolicy`